components/speedreader/resources/panel/components/lists/index.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react'
+
+import * as S from './style'
+import fontSerifSvg from '../../svg/fontSerif'
+import fontSansSvg from '../../svg/fontSans'
+import fontMonoSvg from '../../svg/fontMono'
+import fontDyslexicSvg from '../../svg/fontDyslexic'
+import contentTextOnlySvg from '../../svg/contentTextOnly'
+import contentTextWithImagesSvg from '../../svg/contentTextWithImages'
+
+const fontStyleOptions = [
+  {
+    title: 'Sans',
+    svgIcon: fontSansSvg
+  },
+  {
+    title: 'Serif',
+    svgIcon: fontSerifSvg
+  },
+  {
+    title: 'Mono',
+    svgIcon: fontMonoSvg
+  },
+  {
+    title: 'Dyslexic',
+    svgIcon: fontDyslexicSvg
+  }
+]
+
+const contentStyleOptions = [
+  {
+    title: 'Text with images',
+    svgIcon: contentTextWithImagesSvg
+  },
+  {
+    title: 'Text only',
+    svgIcon: contentTextOnlySvg
+  }
+]
+
+type OptionType = {
+  isSelected: boolean
+  children: JSX.Element
+  onClick?: Function
+  ariaLabel?: string
+}
+
+type ListBoxProps = {
+  children: JSX.Element | JSX.Element[]
```

Comment:
AFAIK it's more encompassing to use `React.ReactChild | React.ReactChild[]` (as it includes strings, numbers, etc), or more simply `React.ReactNode`, but even simpler just use `React.PropsWithChildren<ListBoxProps>` (although in this case it would just be `React.PropsWithChildren<{}>`)

Reply:
added